### DISCUSSION TITLE
How does CPU boost settings work exactly?

### DISCUSSION THREAD

**Discussion by brambedkar59** [seerge/g-helper#921] [General]
Hi, I have few questions.

1. What does CPU boost actually change like TDP, Speed Shift EPP, something else or a combination?
2. Do they work on all devices in the same way? (Mine is Tuf F17 FX706HE 11400H). AC only had 4 power plans: Windows, Silent, Performance and Turbo with no way to change TDP or any other settings on this device.
3. What are ASUS defaults? Does G-helper set them to default when you press "factory defaults" in Fan+Power page?

Sorry if this has already been asked, I ran a search and couldn't find all of the answers.
Thank you.

**Comment by IceStormNG** [comment]
1. The turbo boost setting adjusts the windows power plan and might or might not work depending on your CPU and whether it uses Hardware P-States. Not all settings work on all CPUs. On my Intel machine the only change that I noticed was when I set it to disabled. You can read about this setting on the Microsoft documentation (they talk about Windows server but it works the same on desktop OS). [PERFBOOSTMODE](https://learn.microsoft.com/en-us/windows-server/administration/performance-tuning/hardware/power/power-performance-tuning#processor-performance-boost-mode)

2. On Intel machines, the TDP settings do not work as reliable as on AMD machines. For TDP settings on Intel, better use Throttlestop. It can set the TDP for all modes. The 3 modes still impose hardware limitations in terms of power draw set by the EC.

3. Yes. If you set it to factory defaults, the ASUS defaults are applied. The settings are stored in the firmware and GHelper signals the firmware to switch the profile.

**Reply by brambedkar59** [reply]
1. Looks like it's same on my machine as well. Only "disabled" has any effect, which completely stops CPU from going Turbo.

2. Already using it.

3. Clicking on factory default doesn't do anything for CPU boost but only resets Fan profile.

Thanks for explaining it.
